src/test/java/TestLIC1.java

```diff
@@ -0,0 +1,64 @@
+import org.junit.Assert;
+import org.junit.Test;
+
+public class TestLIC1 {
+
+    @Test
+    public void testInRadius() {
+        // There exists one set of three consecutive data points that cannot all be contained within
+        // a circle of radius1
+        Parameters params = new Parameters(0, 0, 10, 0,
+                0, 0, 0, 0, 0, 0, 0, 0,
+                0, 0, 0, 0, 0, 0, 0);
+        Point a = new Point(0, 0);
+        Point b = new Point(1, 1);
+        Point c = new Point(2, 2);
+        Point[] pts = {a, b, c};
+        Assert.assertEquals(Decide.LIC1(pts, params), false);
+        Assert.assertNotEquals(Decide.LIC1(pts, params), true);
```

Comment:
These assertions test the same thing?
Use `Assert.assertFalse()`?

Reply:
Fixed!